Dockerfile.dapper

```diff
@@ -31,6 +31,7 @@ RUN if [ "${ARCH}" == "amd64" ]; then \
 
 # install controller-tools for crd generation
 RUN go install sigs.k8s.io/controller-tools/cmd/controller-gen@v0.12.0
+RUN wget -q https://github.com/mikefarah/yq/releases/download/v4.34.1/yq_linux_${ARCH}.tar.gz -O - | tar xz && mv yq_linux_${ARCH} /usr/bin/yq
```

Comment:
Nit:

If you can list the YQ version as an env var above (like we do for helm unit test/helm/kustomize) that would be good. It makes it easier for our versioning automation to pick up and do automatic bumps of. If you have a second I would appreciate you doing that for the controller-gen utility in the line before this one as well, I missed that during that PR review.